.gitignore

```diff
@@ -12,4 +12,5 @@ notUsed
 
 taptest.sh
 run.sh
-.vscode
\ No newline at end of file
+.vscode
+get_signatures.sh
```

Comment:
This needs to be removed when the final PR is created for merging to the pgRouting repository.

Reply:
sure